docs/lisk-tools/node-providers.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: API Providers
-slug: /lisk-tools/api-providers
+title: Node Providers
+slug: /lisk-tools/node-providers
```

Comment:
This generates the following error! Please correct.
<img width="1201" alt="image" src="https://github.com/user-attachments/assets/45973d51-7f61-45a2-b768-92a5bc62dc7e" />